zebrad/src/components/mempool/storage/verified_set.rs

```diff
@@ -148,7 +149,7 @@ impl VerifiedSet {
                 .map(|tx| tx.clone().eviction_weight())
                 .collect();
 
-            let dist = WeightedIndex::new(weights).unwrap();
+            let dist = WeightedIndex::new(weights).expect("iterator is not empty");
```

Comment:
It's a bit more complicated than that:
https://docs.rs/rand/latest/rand/distributions/struct.WeightedIndex.html#method.new

```suggestion
            let dist = WeightedIndex::new(weights).expect("there is at least one weight and all weights are valid");
```